enip_backend/ingest/apapi.py

```diff
@@ -0,0 +1,52 @@
+from elex.api.models import Election
+import os
+import io
+import csv
+from datetime import datetime, timedelta, timezone
+
+from ..enip_common.config import AP_API_KEY, INGEST_TEST_DATA
+
+ELECTION_DATE = "2020-11-03"
+OFFICE_IDS = ["P", "S", "H"]
+
+
+def ingest_ap(cursor, ingest_id):
+    # Make the API request
+    election = Election(
+        testresults=INGEST_TEST_DATA,
+        results_level="ru",
```

Comment:
Just a heads-up, it looks like the only way to get the presidential vote by congressional district for NE and ME is to pass "district" here -- so we'll need to somehow do that, I think?